src/accessibility/context_menu/MenuItem.tsx

```diff
@@ -31,15 +30,15 @@ export const MenuItem: React.FC<IProps> = ({ children, label, tooltip, ...props
     const ariaLabel = props["aria-label"] || label;
 
     if (tooltip) {
-        return <AccessibleTooltipButton {...props} role="menuitem" tabIndex={-1} aria-label={ariaLabel} title={tooltip}>
+        return <RovingAccessibleTooltipButton {...props} role="menuitem" aria-label={ariaLabel} title={tooltip}>
             { children }
-        </AccessibleTooltipButton>;
+        </RovingAccessibleTooltipButton>;
     }
 
     return (
-        <AccessibleButton {...props} role="menuitem" tabIndex={-1} aria-label={ariaLabel}>
+        <RovingAccessibleButton {...props} role="menuitem" aria-label={ariaLabel}>
             { children }
-        </AccessibleButton>
+        </RovingAccessibleButton>
```

Comment:
TODO: Apply `RovingTabIndex` changes to 

 - [ ] `MenuItemCheckbox`
 - [ ] `MenuItemRadio`
 - [ ] `StyledMenuItemCheckbox`
 - [ ] `StyledMenuItemRadio`

Reply:
Done in https://github.com/matrix-org/matrix-react-sdk/pull/7353